src/components/progress-tracker/progress-tracker.component.js

```diff
@@ -11,6 +11,9 @@ import {
   StyledValue,
 } from "./progress-tracker.style";
 import useResizeObserver from "../../hooks/__internal__/useResizeObserver";
+import Logger from "../../__internal__/utils/logger";
+
+let deprecatedForwardRefWarnTriggered = false;
```

Comment:
can we change the name of this variable? The name should refer to `orientation` and/or `direction`, not `forwardRef` which is irrelevant here.

Reply:
Yeah sorry copy paste 🤦